src/components/accessories/patientTherapy/PatientTherapy.tsx

```diff
@@ -24,18 +26,22 @@ import checkIcon from "../../../assets/check-icon.png";
 
 const PatientTherapy: FC<TProps> = ({
   createTherapy,
+  getTherapiesByPatientId,
   createTherapyReset,
   isLoading,
   hasSucceeded,
   hasFailed,
+  getMedicals,
 }) => {
   const { t } = useTranslation();
   const infoBoxRef = useRef<HTMLDivElement>(null);
   const [shouldResetForm, setShouldResetForm] = useState(false);
   const [shouldUpdateTable, setShouldUpdateTable] = useState(false);
   const [activityTransitionState, setActivityTransitionState] =
     useState<TherapyTransitionState>("IDLE");
-  const patient = useSelector(
+
+  //There is some "mistake" in apis model about the patient id
+  const patientId = useSelector(
```

Comment:
If the patientId variable contains the patient data, then rename it in patientData in order to improve the semanticity.

Reply:
Or extract only the patient.code and leave the name as before.

Reply:
It is right. But I will still have this syntaxe

`const onSubmit = (therapy: TherapyRowDTO) => {
    setShouldResetForm(false);
    therapy.patID = patientData; // there is some mistake in api data model naming
    createTherapy(therapy);
  };`

because of some mistake in endpoint data model.

Reply:
It's ok as long as we won't fix the APIs.